frontend/packages/ceph-storage-plugin/src/components/bucket-class/create-bc.tsx

```diff
@@ -99,6 +104,34 @@ const CreateBucketClass: React.FC<CreateBCProps> = ({ match }) => {
             },
           };
           break;
+        case NamespacePolicyType.CACHE:
+          payload = {
+            apiVersion: apiVersionForModel(NooBaaBucketClassModel),
```

Comment:
No `apiGroup` required?

Reply:
nope not required..... the method extracts group plus version.